pkg/api/types.go

```diff
@@ -189,6 +189,9 @@ type VolumeSource struct {
 	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
 	// kubelet's host machine and then exposed to the pod.
 	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"gcePersistentDisk"`
+	// AWSPersistentDisk represents a AWS EBS disk that is attached to a
+	// kubelet's host machine and then exposed to the pod.
+	AWSPersistentDisk *AWSPersistentDiskVolumeSource `json:"awsPersistentDisk"`
```

Comment:
OK, I'll change it.  Avoiding another rebase is more important to me than nomenclature :-)